semantic-conventions/src/opentelemetry/semconv/model/semantic_convention.py

```diff
@@ -244,10 +244,55 @@ def __init__(self, group):
         self.members = UnitMember.parse(group.get("members"))
 
 
-class MetricSemanticConvention(BaseSemanticConvention):
+class MetricGroupSemanticConvention(BaseSemanticConvention):
+    GROUP_TYPE_NAME = "metric_group"
+
+
+class MetricSemanticConvention(MetricGroupSemanticConvention):
     GROUP_TYPE_NAME = "metric"
 
-    allowed_keys = ()
+    allowed_keys: Tuple[str, ...] = BaseSemanticConvention.allowed_keys + (
+        "metric_name",
+        "unit",
+        "instrument",
+    )
+
+    yaml_to_markdown_instrument_repr: Dict[str, str] = {
```

Comment:
Ideally, this should be placed in the markdown generator, if it's really about the markdown representation only. Or name it differently, like `canonical_instrument_name_by_yaml_name`

Reply:
I've changed the name to the one you suggested. I've also avoided storing the `self.instrument_markdown_fmt` and instead just calculate it using this dict during the mardkown rendering. 

I think keeping this dict makes sense, because it also defines the allowed yaml values, while mapping them to their markdown values. Otherwise I would have to redefine this whole dict again in the markdown renderer.